thewalrus/quantum/gaussian_checks.py

```diff
@@ -157,3 +157,34 @@ def fidelity(mu1, cov1, mu2, cov2, hbar=2, rtol=1e-05, atol=1e-08):
     # as opposed to 0.25 in Brask. This is because this function returns the square
     # of their fidelities.
     return f
+
+
+def is_symplectic(S, rtol=1e-05, atol=1e-08):
+    """Checks if the matrix is symplectic
+    Args:
```

Comment:
```suggestion
    """Checks if the matrix is symplectic

    Args:
```

Reply:
Here, new lines are needed before both `Args:` and `Returns:` for it to render correctly :)